src/test/java/core/basesyntax/service/impl/CsvFileReaderServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvFileReaderServiceImplTest {
+    private static final String VALID_FILE_PATH = "src/test/resources/file.csv";
+    private static CsvFileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void setUp() {
+        fileReaderService = new CsvFileReaderServiceImpl();
+    }
+
+    @Test
+    public void fileReader_filePathRight_Ok() {

```

Comment:
```suggestion
    public void readFromFile_filePathRight_ok() {
```
the same for other test methods, the first part is the method that you test